src/node_file.cc

```diff
@@ -533,17 +534,25 @@ Local<Value> BuildStatsObject(Environment* env, const uv_stat_t* s) {
 // comes from not creating Error objects on failure.
 static void InternalModuleReadFile(const FunctionCallbackInfo<Value>& args) {
   Environment* env = Environment::GetCurrent(args);
-  uv_loop_t* loop = env->event_loop();
 
   CHECK(args[0]->IsString());
   node::Utf8Value path(env->isolate(), args[0]);
 
+  MaybeLocal<String> content = InternalModuleReadFile(env, *path);
+  if (!content.IsEmpty()) {
+    args.GetReturnValue().Set(content.ToLocalChecked());
+  }
+}
+
+MaybeLocal<String> InternalModuleReadFile(Environment* env, const char* path) {
+  uv_loop_t* loop = env->event_loop();
```

Comment:
@bnoordhuis It also uses `env->isolate()` for creating the returned `String`… I could split that up into separate arguments but I’m not sure it’s worth it